hoomd/md/pytest/test_methods.py

```diff
@@ -138,6 +223,97 @@ def test_langevin_attributes_attached(simulation_factory,
     langevin.tally_reservoir_energy = True
     assert langevin.tally_reservoir_energy
 
+def test_langevin_rattle_attributes():
```

Comment:
This test is nearly identical to the corresponding Brownian test. Could you write a single function named `rattle_attributes_test` (note the naming with `_test` at the end instead of the beginning so that it won't be discovered by `pytest` as a test) that accepts an integrator, then run the same test? You would need to make them use the same manifold, seed, etc, but that would reduce the duplication here and make it easier to add similar tests in the future.